test/Ocelot.ManualTest/Startup.cs

```diff
@@ -37,7 +37,7 @@ public void ConfigureServices(IServiceCollection services)
                 })
                 .WithDictionaryHandle();
             };
-
+            services.AddMemoryCache();
```

Comment:
If this is required for rate limiting to work maybe we should add it in ServiceCollectionExtensions.AddOcelot